docs/virtual_machines/resources_requests_and_limits.md

```diff
@@ -52,8 +69,23 @@ metadata:
 
 ### Memory limits on the container
 - By default, no memory limit is set on the container
+- If auto memory limits is enabled (see next section), then the container will have a limit of 2x the requested memory.
 - Manually setting a memory limit on the VM(I) will set the same value on the container
 
 #### Warnings
 - Memory limits have to be more than memory requests + overhead, otherwise the container will have memory requests > limits and be rejected by Kubernetes.
 - Memory usage bursts could lead to VM crashes when memory limits are set
+
+
+### Auto memory limits
+KubeVirt provides a feature gate(`AutoResourceLimitsGate`) to automatically set memory limits on VM(I)s.
+By enabling this feature gate, memory limits will be added to the vmi if all the following conditions are true:
+
+- The namespace where the VMI will be created has a ResourceQuota containing memory limits.
+- The VMI has no manually set memory limits.
+- The VMI is not requesting dedicated CPU.
```

Comment:
If I request dedicated CPU, can't I set memory limits? can you please elaborate on the connection between the two?

Reply:
Sure! Here I just wanted to show that the auto memory limits feature will not take action if the vmi is requesting the dedicatedCPU. This because when dedicatedCPU is true, Kubevirt will create a virt-launcher pod with Guaranteed QoS, by set the memory limits equals the requests.
IOW auto memory limits will not take action, because the dedicatedCPU will already set its own memory limits.
Does it help?

Reply:
@fossedihelm Sure! thanks for the information